File: 1365-HowManyNumbersAreSmallerThantheCurrentNumber/1365-HowManyNumbersAreSmallerThantheCurrentNumber.py
            if sorted_nums[i+1] < curr_num:
                remaining_val = len(sorted_nums) - (i+1)
                dict[curr_num] = remaining_val
        
        dict[sorted_nums[-1]] = 0

        result = []
        for num in nums:
            result.append(dict[num])

        return result

[

File: 1365-HowManyNumbersAreSmallerThantheCurrentNumber/1365-HowManyNumbersAreSmallerThantheCurrentNumber.java
                int remainingVal = i;
                dict.put(currNum, remainingVal);
        for (int i = sortedNums.length - 1; i > 0; i--) {
            int currNum = sortedNums[i];
            if (sortedNums[i - 1] < currNum) {
        Arrays.sort(sortedNums);

        Map<Integer, Integer> dict = new HashMap<>();
class Solution {
    public int[] smallerNumbersThanCurrent(int[] nums) {
        int[] sortedNums = nums.clone();
[
